serial_windows.go

```diff
@@ -130,7 +131,83 @@ func (p *Port) Read(buf []byte) (int, error) {
 // Discards data written to the port but not transmitted,
 // or data received but not read
 func (p *Port) Flush() error {
-	return purgeComm(p.fd)
+	err := purgeComm(p.fd)
+	clearCommError(p.fd)
+	return err
+}
+
+/*
+This function was taken with minor modifications from the go.bug.st/serial package (https://github.com/bugst/go-serial), and is subject to the conditions of its license (reproduced below):
+
+Copyright (c) 2014, Cristian Maglie.
+All rights reserved.
+
+Redistribution and use in source and binary forms, with or without
```

Comment:
This is almost the standard Go License, which tarm/serial is already under.  The Go license is a 3 clause BSD license like this one Copyright the Go Authors.   I would like to keep the existing license text and not add this.

Unfortunately @cmaglie is not in the standard Go AUTHORS file right now:
https://golang.org/AUTHORS